sql/core/src/test/scala/org/apache/spark/sql/DynamicPartitionPruningSuite.scala

```diff
@@ -140,9 +147,11 @@ abstract class DynamicPartitionPruningSuiteBase
       .format(tableFormat)
       .saveAsTable("code_stats")
 
-    sql("ANALYZE TABLE fact_stats COMPUTE STATISTICS FOR COLUMNS store_id")
-    sql("ANALYZE TABLE dim_stats COMPUTE STATISTICS FOR COLUMNS store_id")
-    sql("ANALYZE TABLE code_stats COMPUTE STATISTICS FOR COLUMNS store_id")
+    if (withColumnStats) {
```

Comment:
We don't support ANALYZE TABLE commands for v2 tables. That's why a couple of tests are expected to fail.